### PR TITLE
Frontend Student Page Complete

### PR DESCRIPTION
**Pull request by themegatree**
AT-16: Students page, listing all students

### CODE REVIEW

File: src/components/Students/Student.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const Student = (props) => {
+  console.log(props.index)
```
Comment (dearshrewdwit) [inline src/components/Students/Student.js:4]: remove debugging statement

File: src/components/Students/Student.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  return (
+
```
Comment (dearshrewdwit) [inline src/components/Students/Student.js:6]: remove whitespace

File: src/components/Students/Students.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    super(props)
+    this.state = {
+      students: []
+    }
+  }
+
+  componentDidMount() {
+    fetch(`${process.env.REACT_APP_API_URL}/api/students`)
+      .then(res => res.json())
+      .then(data => this.setState({students: data.students.sort((a,b)=> (a.lastName > b.lastName) ? 1: -1)}));
```
Comment (dearshrewdwit) [inline src/components/Students/Students.js:16]: Let's extract this to a function to help with clarity (is it DESC?)

```js
sortByLastNameDesc(students) {
  return students.sort((a,b) => (a.lastName > b.lastName) ? 1 : -1
}

// in other func
this.setState({students: this.sortByLastName(data.students)}
```

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]
Nice

**Review by dearshrewdwit — DISMISSED** [review]

**Review by dearshrewdwit — APPROVED** [review]

**Review by RyanVanDijck — APPROVED** [review]
Looks fine to me.

**Pull request merged**